setup.py

```diff
@@ -85,13 +85,20 @@ def get_version(version_file):
     ],
     python_requires=">=3.6",
     # What does your project relate to?
-    keywords="skeleton",
+    keywords="Technical Payload Testing (TPT) Report",
     packages=find_packages(where="src"),
     package_dir={"": "src"},
-    package_data={"tpt_reports": ["data/*.txt"]},
+    package_data={"tpt_reports": ["fonts/*.ttf", "assets/*.png"]},
```

Comment:
Alphabetize for ease of future maintenance:
```suggestion
    package_data={"tpt_reports": ["assets/*.png", "fonts/*.ttf"]},
```